tests/BUILD.bazel

```diff
@@ -199,6 +199,7 @@ go_test(
         "//tests/hotplug:go_default_library",
         "//tests/infrastructure:go_default_library",
         "//tests/launchsecurity:go_default_library",
+        "//tests/libconfigmap:go_default_library",
```

Comment:
```
tests/utils.go: remove unused function
Signed-off-by: bmordeha <bmordeha@redhat.com>
```
can you have a clearer commit message?

I see that you remove here a function, but you also add code and you don't explain why.

Reply:
Sorry about that. I added the changes to the wrong commit.
I fixed it